_episodes/02-introduction-to-agile.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Introduction to agile"
+teaching: 10
+exercises: 20
+questions:
+- "What is 'agile'?"
+- "Why should I work in an agile way?"
+objectives:
+- "List drawbacks of Waterfall model"
+- "Describe Agile Manifesto"
+- "List benefits of following Agile manifesto"
+keypoints:
+- "The agile manifesto provides good pointers for successfully working on software projects as a team"
+---
+# Waterfall model
+Typically, many real-world (software development) projects
+are divided into different phases, where each phase depends on the
+output of the previous phase. We refer to this as the 'waterfall model',
+because each phase flows into the next phase like a waterfall:
+
+![Waterfall model](../fig/waterfall-model.png)
+
+The key is that there is a clear project timeline and agreed-upon deliverables
+at the beginning of the project.
+
+> ## Discussion
+> What do you think are potential drawbacks of the waterfall model?
+>
+> > ## Solution
+> > - Less customer involvement (only in the beginning)
+> > - Users usually only really know the requirements once they have working software, but then it is too late.
```

Comment:
Moreover: during the development time the requirements of the software may change, but in waterfall there is no way to incorporate these changes. So you end up with software that does not do what the user needs

Reply:
Good one, I added it!